web/packages/design/src/Alert/Alert.tsx

```diff
@@ -302,6 +302,7 @@ const IconContainer = styled.div<{ kind: AlertKind }>`
   border-radius: 50%;
   line-height: 0;
   margin-right: ${p => p.theme.space[3]}px;
+  align-self: flex-start;
```

Comment:
> Top-align the icon in alerts with large amounts of vertical content.

I wanted to do the same thing, but I don't think you can just change `IconContainer` like this, as it'll affect all alerts across the whole app. Those that are supposed to be on a single line are going to look off (e.g. https://localhost:9002/?path=/story/design-alerts--actionable).

This needs to be configurable on a per-alert basis.

Reply:
My solution keeps the original placement of the icon for other alerts.

| Zac's | Rafał's |
| --- | --- |
| <img width="1119" alt="zacs" src="https://github.com/user-attachments/assets/3a0e9332-7129-4210-a0ca-30827079a871" /> | <img width="1119" alt="rafals" src="https://github.com/user-attachments/assets/2dec2c9f-fcde-46ae-893e-304882781fa2" /> |

I added `alignItems` which allows you to align both the icon and the action buttons to the top. I'm going to need it in VNet diagnostics anyway. Would using it work for you in Discover?

Reply:
Oops, I didn't know you talked with Bartosz about this. I still think it might be best to change only the Discover alerts for now?

Reply:
Your solution seems fine, but I am confused at the state of things now that you've pushed to my branch.

Are we good to go now on this issue or are more changes required?

Reply:
You'd need to add `alignItems="flex-start"` to relevant alerts in Discover where the icon should be aligned to the top. I suppose every warning that used to use `HintBox` should now be `<Warning alignItems="flex-start">`.